lms/static/scripts/frontend_apps/components/dashboard/AssignmentActivity.tsx

```diff
@@ -31,12 +27,24 @@ export default function AssignmentActivity() {
 
   return (
     <Card>
-      <CardHeader fullWidth>
-        <CardTitle tagName="h2" data-testid="title">
+      <CardHeader fullWidth classes="flex-col !gap-x-0 !items-start">
+        {assignment.data && (
+          <div className="mb-3 mt-1 w-full">
+            <DashboardBreadcrumbs
+              links={[
+                {
+                  title: assignment.data.course.title,
+                  href: `/courses/${assignment.data.course.id}`,
```

Comment:
This path should be escaped.

Reply:
I believe we had a function that could be used like this

    encodeURI`/some/uri/${with}/params`

But I can't find it now, so I have just used `/courses/${encodeURIComponent(assignment.data.course.id)}`

Reply:
Scratch that. I have just found `urlPath` 😅